cluster/src/main/java/org/apache/iotdb/cluster/server/member/DataGroupMember.java

```diff
@@ -1369,6 +1370,24 @@ public void getChildNodePathInNextLevel(Node header, String path,
     }
   }
 
+  @Override
+  public void getAllMeasurementSchema(Node header, String path,
+      AsyncMethodCallback<List<List<String>>> resultHandler) throws TException {
+    if (!syncLeader()) {
+      resultHandler.onError(new LeaderUnknownException(getAllNodes()));
+      return;
+    }
+    try {
+      List<List<String>> res = new ArrayList<>();
+      for(String[] element : MManager.getInstance().getAllMeasurementSchema(path)){
+        res.add(Arrays.asList(element));
+      }
+      resultHandler.onComplete(res);
```

Comment:
The current implementation cannot avoid the problem of sending duplicated metadata.
The solution may be modifying the interfaces in MManager to add a filter that will skip the unsatisfying timeseries that do not belong to the slots managed by the member.
It is a little tricky and it is okay if you do it in another PR.